src/frontend-pwa/src/components/utility/SplashScreen/SplashScreen.tsx

```diff
@@ -20,17 +20,24 @@ export default function SplashScreen() {
     setAppData,
     initializeEulaState,
     updateSettings,
+    state,
   } = useAppService();
 
   useEffect(() => {
```

Comment:
Id recommend a comment above this useEffect to explain what it's doing

Reply:
@BradyMitch Good call!